crates/trigger-redis/src/lib.rs

```diff
@@ -210,13 +210,8 @@ impl<F: RuntimeFactors> Subscriber<F> {
             .instantiate(())
             .await?;
 
-        let guest = {
-            let exports = &mut instance.exports(&mut store);
-            let mut inbound_redis_export = exports
-                .instance("fermyon:spin/inbound-redis")
-                .context("no fermyon:spin/inbound-redis instance found")?;
-            inbound_redis::Guest::new(&mut inbound_redis_export)?
-        };
+        let guest_indices = inbound_redis::GuestIndices::new_instance(&mut store, &instance)?;
```

Comment:
I'm not familiar with this `GuestIndices` API in wasmtime, and I'm having a hard time understanding why this is used over the `Guest::new` constructor. Can you explain this a bit?

Reply:
The short answer is that `Guest::new` doesn't exist any more. The longer answer is that Wasmtime's now refactored to better support exported interfaces at different versions than listed in the WIT (e.g. automatically handling 0.2.1-but-you-gave-me-0.2.0 and situations like that). That refactoring ended up moving some things around and this is the way this is loaded now.

The other slightly long-answer bit is that most of the bindings are intended to operate at the world level but this is operating at the individual interface level which means that the APIs used here are somewhat non-standard in the sense that the top-level world bits that `bindgen!` generates has all the helpers and individual interfaces are left to pull the pieces together individually. Perhaps something I can help refactor within Spin in the future?